sys/amd64/amd64/pmap.c

```diff
@@ -3933,6 +3933,12 @@ pmap_extract_and_hold(pmap_t pmap, vm_offset_t va, vm_prot_t prot)
 	return (m);
 }
 
+/*
+ *	Routine:	pmap_kextract
+ *	Function:
+ *		Extract from the kernel page table the physical address
```

Comment:
Can this be made to more closely follow the wording for pmap_extract? Something more like:

> Extract the physical page address associated with the given kernel virtual address.